build.sbt

```diff
@@ -306,6 +323,22 @@ def nativeImageDockerSettings: Seq[Setting[_]] = dockerSettings ++ Seq(
   }
 )
 
+def assemblySettings(jarName: String) =
```

Comment:
There was lots of code duplication related to assembly that I factored out into this utility method.

Reply:
👍